test/e2e/testing-manifests/storage-csi/external-provisioner/rbac.yaml

```diff
@@ -84,6 +93,21 @@ rules:
 - apiGroups: ["coordination.k8s.io"]
   resources: ["leases"]
   verbs: ["get", "watch", "list", "delete", "update", "create"]
+# Permissions for CSIStorageCapacity are only needed enabling the publishing
+# of storage capacity information.
+- apiGroups: ["storage.k8s.io"]
+  resources: ["csistoragecapacities"]
+  verbs: ["get", "list", "watch", "create", "update", "patch", "delete"]
+# The GET permissions below are needed for walking up the ownership chain
+# for CSIStorageCapacity. They are sufficient for deployment via
+# StatefulSet (only needs to get Pod) and Deployment (needs to get
+# Pod and then ReplicaSet to find the Deployment).
```

Comment:
does this mean that pods created from job / cronjob / replicationcontroller / custom-pod-spec-containing objects don't work properly with CSIStorageCapacity?

Reply:
The RBAC rules need to be adapted depending on how the external-provisioner gets deployed. This is basically just an example of how to do it, not necessarily what everyone has to use.